test/e2e/apps/replica_set.go

```diff
@@ -35,7 +36,37 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-func newRS(rsName string, replicas int32, rsPodLabels map[string]string, imageName string, image string) *extensions.ReplicaSet {
+func newRS(rsName string, replicas int32, rsPodLabels map[string]string, imageName string, image string) *apps.ReplicaSet {
```

Comment:
@kow3ns mentioned the e2e test should used the untyped client so it should theoretically work on both old and new e2e clusters.